storybook/.eslintrc.cjs

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
Since we're working on setting up `biome` as linter and formatter we could probably remove `eslint` from this PR?